tracing/trace_entries.py

```diff
@@ -354,7 +354,7 @@ def load_entries(self, entries):
 
             if 'nw_proto' in ip_:
                 self.nw_proto = ip_['nw_proto']
-                if 'tp' not in trace:
+                if ((ip_['nw_proto'] == 17 or ip_['nw_proto'] == 6) and 'tp' not in trace):
```

Comment:
Let's use a variable here instead of hardcoding these protocol values, there's a module already defining these const values https://github.com/kytos-ng/sdntrace/blob/master/constants.py#L10-L11